Lombiq.Tests.UI.Samples/Tests/MonkeyTests.cs

```diff
@@ -66,9 +66,9 @@ public MonkeyTests(ITestOutputHelper testOutputHelper)
             browser,
             configuration =>
                 // This is necessary to work around this bug: https://github.com/OrchardCMS/OrchardCore/issues/11420.
-                configuration.AssertBrowserLog = messages => messages.ShouldNotContain(
+                configuration.AssertBrowserLog = logEntries => logEntries.ShouldNotContain(
```

Comment:
`logEntries` is better because it refers to the type(`IEnumerable<LogEntry>`), but it differs from the cases where similar type(`LogEntry`) is used but the variable is called `message`, so I suggest using `messages` here, or `logEnry` - would better - in other cases.
```suggestion
                configuration.AssertBrowserLog = messages => messages.ShouldNotContain(
```
What is your opinion?

Reply:
The singular should also be `logEntry` here. I didn't leave it like that deliberately, just didn't notice it being used in this class too, so fixing that.